tests/manager/api/util/test_xray.py

```diff
@@ -1,17 +1,21 @@
-from unittest.mock import MagicMock, call
+from unittest.mock import MagicMock, call, patch
+
+import pytest
```

Comment:
These test changes could be broken out to separate PR. They are necessary because we now run the tests against a container that has a version set. The tests previously assumed this was not the cause, causing some of these tests to fail. 

The changes just mock the `__version__` variable so it doesn't matter if its set or not, the tests will always behave correctly.